contracts/Profile.sol

```diff
@@ -92,6 +107,46 @@ contract Profile {
     //     profileStorage.setNodeAddress(identityId, hashFunctionId);
     // }
 
+    // TODO: Define where it can be called, change internal modifier
+    function _setAvailableNodeAddresses(uint72 identityId) internal {
+        ProfileStorage ps = profileStorage;
+        HashingProxy hp = hashingProxy;
+
+        UnorderedIndexableContractDynamicSetLib.Contract[] memory hashFunctions = hp.getAllHashFunctions();
+        uint256 hashFunctionsNumber = hashFunctions.length;
+        for (uint8 i; i < hashFunctionsNumber; ) {
+            ps.setNodeAddress(identityId, hashFunctions[i].id);
+            unchecked { i++; }
+        }
+    }
+
+    function stakeReward(uint72 identityId) external onlyAdmin(identityId) {
```

Comment:
lets rename the function to stakeAccumulatedOpFee (so its clearer). technically these are not rewards

Reply:
Done